### DISCUSSION TITLE
24h time format for date/time picker

### DISCUSSION THREAD

**Discussion by tobehn** [Budibase/budibase#2765] [Ideas]
As a User i want to be able to set a timepickers time selection to 24h time format and thereby be able to select hour values greater than 12.

![image](https://user-images.githubusercontent.com/40893107/135036704-f874d9ac-25ca-41b7-b039-14c68fd51c10.png)

Perhaps even a higher level setting in the system could regulate it. Like in the main application settings time format 12/24h in order to convert timestamps system wide.

I just realized that Auto Columns (i.e. Created At) are using 24h time format.
![image](https://user-images.githubusercontent.com/40893107/135037359-581072a5-240b-4d46-9cf8-da184d3c3d52.png)

Also the time format in time fields is 24h

**So it is just about the input field - to have an option to set a 24h formatted time.**

**Comment by mjashanks** [comment]
Yes - general date formatting is needed. Even for different Regions.
